src/pages/[lang]/assistenza.tsx

```diff
@@ -1,226 +1,108 @@
-                textDecoration: "none",
-                color: "#17324D",
-              }}
-            >
-              {PAGOPA_HELP_EMAIL}
-            </Link>
-            .
-          </Typography>
-          <Button
-            variant="outlined"
-            color="primary"
-            component="a"
-            href={"mailto:" + PAGOPA_HELP_EMAIL}
-          >
-            Scrivici
-          </Button>
-        </Stack>
-      </Grid>
-    </Grid>
-  );
-};
+  return { props: { translations, lang: params.lang } };
+}
 
 const Assistenza: NextPage = () => {
+  const { t } = useTranslation(["common", "assistenza"]);
   const [currentTab, setCurrentTab] = useState({ index: 0, visible: true });
   const transitionDuration = 500;
   const containerRef = useRef(null);
-  const tabsData = getAssistenzaTabsData("tabs assistenza 1");
-  const headingTitleData = getAssistenzaHeadingTitleData(
-    "heading title assistenza 1"
-  );
+
+  const tabs = [
+    t("tab.1.title", { ns: "assistenza" }),
+    t("tab.2.title", { ns: "assistenza" }),
+  ];
+
+  const cardsContent: Array<AssistanceCardsProps> = [
+    {
+      // cards cittadini e imprese
+      cards: [
+        {
+          title: t("tab.1.card.1.title", { ns: "assistenza" }),
+          href: "/faq#send-come-accedere",
+          text: t("tab.1.card.1.cta", { ns: "assistenza" }),
+        },
+        {
+          title: t("tab.1.card.2.title", { ns: "assistenza" }),
+          href: "/faq#documenti-aar",
+          text: t("tab.1.card.2.cta", { ns: "assistenza" }),
+        },
+        {
+          title: t("tab.1.card.3.title", { ns: "assistenza" }),
+          href: "/faq#perfezionamento-cosa-significa",
+          text: t("tab.1.card.3.cta", { ns: "assistenza" }),
+        },
+      ],
+      button: {
+        text: t("tab.1.cta", { ns: "assistenza" }),
+        href: "/faq",
+      },
+    },
+    {
+      // cards enti mittenti
+      cards: [
+        {
+          title: t("tab.2.card.1.title", { ns: "assistenza" }),
+          href: "https://docs.pagopa.it/send-faq-enti/argomenti/adesione/chi-puo-aderire",
+          text: t("tab.2.card.1.cta", { ns: "assistenza" }),
+        },
+        {
+          title: t("tab.2.card.2.title", { ns: "assistenza" }),
+          href: "https://docs.pagopa.it/area-riservata-enti-piattaforma-notifiche/area-riservata-enti-send-servizio-notifiche-digitali/processo-di-adesione-a-send",
+          text: t("tab.2.card.2.cta", { ns: "assistenza" }),
+        },
+        {
+          title: t("tab.2.card.3.title", { ns: "assistenza" }),
+          href: "https://docs.pagopa.it/manuale-operativo/piattaforma-notifiche-digitali-manuale-operativo/il-processo-di-notificazione",
+          text: t("tab.2.card.3.cta", { ns: "assistenza" }),
+        },
+      ],
+      button: {
+        text: t("tab.2.cta", { ns: "assistenza" }),
+        href: "/pubbliche-amministrazioni/documenti/",
+      },
+    },
+  ];
 
   const handleResize = () => {

```

Comment:
I don't understand why we need this. Do a check and remove it if it useless

Reply:
if you try removing it and resize the window horizontally you can see the cards having different heights.